paramak/parametric_components/toroidal_field_coil_princeton_d.py

```diff
@@ -4,44 +4,41 @@
 from paramak.utils import add_thickness
 from scipy import integrate
 from scipy.optimize import minimize
+from typing import Optional
```

Comment:
could these imports be sorted (right mouse click in vs code and select sort imports), I can't quite make the change suggestion in the GUI as it needs to go up too many lines